src/basics.md

```diff
@@ -12,6 +12,7 @@
 | [Run an external command passing it stdin and check for an error code][ex-parse-subprocess-input] | [![regex-badge]][regex] | [![cat-os-badge]][cat-os] [![cat-text-processing-badge]][cat-text-processing] |
 | [Run piped external commands][ex-run-piped-external-commands] | [![std-badge]][std] | [![cat-os-badge]][cat-os] |
 | [Redirect both stdout and stderr of child process to the same file][ex-redirect-stdout-stderr-same-file] | [![std-badge]][std] | [![cat-os-badge]][cat-os] |
+| [Continuously process child process' outputs][ex-continuous-process-output] | [![std-badge]][std] | [![cat-os-badge]][cat-os] |
```

Comment:
hmm text-processing badge might make sense. What are your thoughts @ludwigpacifici ?

Reply:
Yep. Plus it will be homogeneous with similar recipes like [Run an external command and process stdout](https://rust-lang-nursery.github.io/rust-cookbook/basics.html#ex-parse-subprocess-output).